python/hsfs/util.py

```diff
@@ -143,12 +142,21 @@ def get_hudi_datestr_from_timestamp(timestamp):
 def convert_event_time_to_timestamp(event_time, time_zone=timezone.utc):
```

Comment:
Since the time_zone argument is not used anywhere, I would argue for removing it here and to rely on the timzone information provided with the timestamp/datetime objects. Is there is no timezone information, e.g. when the user provides a string input, we can always set it to UTC.